src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers.csproj

```diff
@@ -1,12 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <XAConfigPath>..\..\bin\Build$(Configuration)\XAConfig.props</XAConfigPath>
+  </PropertyGroup>
+  <Import Condition="Exists ('$(XAConfigPath)')" Project="$(XAConfigPath)" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(DotNetTargetFramework)</TargetFrameworks>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <SignAssembly>true</SignAssembly>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
```

Comment:
The `net8.0` build failed without this:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9093665&view=logs&j=4ab9874f-3b82-50ef-2a6b-4340d0043621&t=1d9063ca-c961-5752-dc23-cb1130ee3d46&l=1141

Which is kind of odd, I thought that was the default in `net5.0`+? We should use it anyway to make incremental builds quicker.